static/app/constants/index.tsx

```diff
@@ -346,6 +356,7 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Profile Hours'),
     productName: t('Continuous Profiling'),
     uid: 17,
+    isBilledCategory: false,
```

Comment:
Should this be true?
If not we need to remember to add this for continuous profiling launch.

Reply:
... otherwise LGTM

Reply:
it would need to be true for GA, for simplicity i think i'll just wait til after launch to fix and merge this PR